### PR TITLE
KP-1163 ([aws-plugin-library] Refactor aws-plugin-library to use kaholo-plugin-library under the hood)

### PR DESCRIPTION
**Pull request by xkcm**
[**KP-1163 in Jira**](https://kaholo.atlassian.net/browse/KP-1163)

### CODE REVIEW

**Review by kenskyfish — APPROVED** [review]
I have faith this will work

File: tests/autocomplete.test.js [hunk 1]
```diff
@@ -345,11 +208,11 @@ describe("filterItemsByQuery", () => {
 
   test("limits the number of returned values to defined default value", () => {
     const veryLongAutocompleteItemsArray = [];
-    for (let i = 0; i < 10000; i++) {
+    for (let i = 0; i < 10000; i += 1) {
```
Comment (kbuchacz) [inline tests/autocomplete.test.js:211]: Why the change?

**Comment by xkcm** [comment]
I'll test this on the AWS EC2 plugin

File: tests/autocomplete.test.js (hunk 1 above)
Comment (xkcm) [reply tests/autocomplete.test.js:211]: eslint complained about ++
